vortex-array/src/stats/statsset.rs

```diff
@@ -320,19 +322,11 @@ impl StatsSet {
     }
 }
 
-impl From<EnumMap<Stat, Option<Scalar>>> for StatsSet {
-    fn from(values: EnumMap<Stat, Option<Scalar>>) -> Self {
-        Self { values }
-    }
-}
-
 impl FromIterator<(Stat, Scalar)> for StatsSet {
     fn from_iter<T: IntoIterator<Item = (Stat, Scalar)>>(iter: T) -> Self {
-        let mut values = EnumMap::<Stat, Option<Scalar>>::default();
-        iter.into_iter().for_each(|(stat, scalar)| {
-            values[stat] = Some(scalar);
-        });
-        Self { values }
+        Self {
```

Comment:
should probably make sure stats are unique

Reply:
you are correct